torch_tvm/operators.cpp

```diff
@@ -216,7 +222,8 @@ RegisterTVMOperator reg({
              bias_add_op, {out, inputs[2]}, tvm::Attrs(bias_add_attrs), {});
        }
        return out;
-     }},
+     },
+     "", PARAM_INDICES(convolution)},
```

Comment:
re my comment below, this would be fine as
```
}, "", {1, 2}},
```

Reply:
So I dont like constant 1s and 2s lying around without explaining what they are. `PARAM_INDICES` conveys clearly what they are. If you think it is overly complicated to go somewhere else and add a define for a new op, I understand that. But note that it is cleaner this way. I would have `op_params.h` where all such defines exist. We can also make that more readable by having comments there.